AtlasStarter/Program.cs

```diff
@@ -0,0 +1,194 @@
+﻿using System;
+using System.Collections.Generic;
+using MongoDB.Driver;
+
+namespace AtlasStarter
+{
+    class MainClassNoPrompt
+    {
+        public static void Main(string[] args)
+        {
+            // TODO:
+            // Replace the placeholder connection string below with your
+            // Altas cluster specifics. Be sure it includes
+            // a valid username and password! Note that in a production environment,
+            // you do not want to store your password in plain-text here.
+
+            var mongoUri = "mongodb+srv://<username>:<password>@<cluster-url>?retryWrites=true&w=majority";
```

Comment:
In our onboarding wizard, we could potentially give them the fully formed URI, complete with their username, password and cluster URL embedded inside.  We could tell them to find this line and swap it out completely.

If this repo were *only* to compliment our wizard, I'd say we should remove the above string and just have 
```suggestion
            var mongoUri = <replace with the URI we give you>;
```
to make things even more explicit.  This would reduce the ease of use for non-wizard users of this repo, though, so probably not a good idea.  Lemme know if you have any ideas along that front, though.

Reply:
I prefer your approach, which is closer to how we do the university courses.